### PR TITLE
sys-var: change `tidb_scatter_region` usage definition

### PR DESCRIPTION
**Pull request by Benjamin2037**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
As title mentioned.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -4149,12 +4149,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值: ``
+- 可选值: ``, `TABLE`, `GLOBAL'
```
Comment (hfxsd) [inline system-variables.md:4157]:
```suggestion
- 可选值: ``, `TABLE`, `GLOBAL`
```

Comment (hfxsd) [inline system-variables.md:4157]:
```suggestion
- 默认值：``
- 可选值：``, `TABLE`, `GLOBAL'
```

File: system-variables.md [hunk 2]
```diff
@@ -4149,12 +4149,13 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。`` 默认值，该选项表示对于表 Region 不做打散，`TABLE` 表示 TiDB 会按照表数据分布情况对于表数据进行打散，'GLOBAL' 表示 TiDB 会按照集群整体的数据分布情况对于数据进行打散。在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
```
Comment (hfxsd) [inline system-variables.md:4158]:
```suggestion
- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
    - 默认值为 ``，表示对于表 Region 不进行打散。
    - `TABLE` 表示 TiDB 会按照表数据分布情况对于表数据进行打散。
    - `GLOBAL` 表示 TiDB 会按照集群整体的数据分布情况对于数据进行打散。在建表语句执行时，同步打散刚分裂出的 Region。
```

File: system-variables.md [hunk 3]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``, `TABLE`, `GLOBAL'
```
Comment (hfxsd) [inline system-variables.md:4157]:
```suggestion
- 可选值：``，`TABLE`，`GLOBAL'
```

File: system-variables.md [hunk 4]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL'
```
Comment (hfxsd) [inline system-variables.md:4157]:
```suggestion
- 可选值：``，`TABLE`，`GLOBAL`
```

File: system-variables.md [hunk 5]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值为 ``，表示对于表 Region 不进行打散。
+    - `TABLE` 表示 TiDB 会按照表数据分布情况对于表数据进行打散。
+    - `GLOBAL` 表示 TiDB 会按照集群整体的数据分布情况对于数据进行打散。在建表语句执行时，同步打散刚分裂出的 Region。
```
Comment (hfxsd) [inline system-variables.md:4161]:
```suggestion
    - `TABLE` 表示 TiDB 会按照表数据分布情况对表数据进行打散。
    - `GLOBAL` 表示 TiDB 会按照集群整体的数据分布情况对数据进行打散。在建表语句执行时，同步打散刚分裂出的 Region。
```

**Review by River2000i — COMMENTED** [review]
/LGTM

**Review by niubell — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值为 ``，表示对于表 Region 不进行打散。
+    - `TABLE` 表示 TiDB 会按照表数据分布情况对表数据进行打散。
+    - `GLOBAL` 表示 TiDB 会按照集群整体的数据分布情况对数据进行打散。在建表语句执行时，同步打散刚分裂出的 Region。
```
Comment (Frank945946) [inline system-variables.md:4161]:
```suggestion
    -   `TABLE` ，表示当用户建表时，设置 PRE_SPLIT_REGIONS 或者 SHARD_ROW_ID_BITS ，预分裂多个 region 的场景下，会对这些表的 region 按表的粒度进行打散。但是如果建表时没有设置这 2 个属性，在快速创建大量表的场景，会导致这些表的 region 集中在其中几个 TiKV 节点上，造成 region 分布不均匀。
    - `GLOBAL` TiDB 会根据整个集群的数据分布情况来打散新建表的 Region。特别是快速创建大量表的时候，可以有效避免 Region 过度集中在少数几个 TiKV 节点上，确保 Region 分布均匀。
```

File: system-variables.md [hunk 7]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 
 ### `tidb_scatter_region`
 
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
```
Comment (River2000i) [inline system-variables.md:4152]: keep the original global level?

File: system-variables.md [hunk 8]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值为 ``，表示对于表 Region 不进行打散。
+    -   `TABLE`：表示当你建表时，如果设置了 `PRE_SPLIT_REGIONS` 或者 `SHARD_ROW_ID_BITS`，预分裂多个 Region 的场景下，会按表的粒度对这些表的 Region 进行打散。但是如果你这建表时没有设置上述属性，在快速创建大量表的场景，会导致这些表的 Region 集中在其中几个 TiKV 节点上，造成 Region 分布不均匀。
```
Comment (jebter) [inline system-variables.md:4160]: 你这建表时-->你这建表时

Comment (hfxsd) [inline system-variables.md:4160]:
```suggestion
    - `TABLE`：表示当你在建表时，如果设置了 `PRE_SPLIT_REGIONS` 或者 `SHARD_ROW_ID_BITS`，预分裂多个 Region 的场景下，会按表的粒度对这些表的 Region 进行打散。但是如果你在建表时没有设置上述属性，在快速创建大量表的场景，会导致这些表的 Region 集中在其中几个 TiKV 节点上，造成 Region 分布不均匀。
```

File: system-variables.md (hunk 7 above)
Comment (Benjamin2037) [reply system-variables.md:4152]: Yes，Current implement should add session level sys var change and also keep global level implementation.

File: system-variables.md [hunk 9]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
-- 作用域：GLOBAL
+- 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
```
Comment (lilin90) [inline system-variables.md:4157]: @hfxsd 后续需要关注下 PR 合并后官方文档的效果，第一个空的可选值导致 GitHub 页面解析有问题：
image

File: system-variables.md [hunk 10]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值为 ``，表示对于表 Region 不进行打散。
+    - `TABLE`：表示在建表时，如果设置了 `PRE_SPLIT_REGIONS` 或者 `SHARD_ROW_ID_BITS`，预分裂多个 Region 的场景下，会按表的粒度对这些表的 Region 进行打散。但是如果在建表时没有设置上述属性，需要快速创建大量表的场景，会导致这些表的 Region 集中在其中几个 TiKV 节点上，造成 Region 分布不均匀。
```
Comment (lilin90) [inline system-variables.md:4160]:
```suggestion
    - `TABLE`：表示在建表时，如果设置了 `PRE_SPLIT_REGIONS` 或者 `SHARD_ROW_ID_BITS` 属性，预分裂多个 Region 的场景下，会按表的粒度对这些表的 Region 进行打散。但是如果在建表时没有设置上述属性，需要快速创建大量表的场景，会导致这些表的 Region 集中在其中几个 TiKV 节点上，造成 Region 分布不均匀。
```

File: system-variables.md [hunk 11]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值为 ``，表示对于表 Region 不进行打散。
```
Comment (lilin90) [inline system-variables.md:4159]:
```suggestion
    - 默认值 ``：表示对于表 Region 不进行打散。
```

File: system-variables.md [hunk 12]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：``
+- 可选值：``，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
+    - 默认值 ``：表示对于表 Region 不进行打散。
```
Comment (lilin90) [inline system-variables.md:4159]: @Benjamin2037 为避免解析显示问题，默认值的写法里可以是否可以加个空格？
```suggestion
- 默认值：` `
- 可选值：` `，`TABLE`，`GLOBAL`
- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
    - 默认值 ` `：表示对于表 Region 不进行打散。
```

File: system-variables.md (hunk 9 above)
Comment (hfxsd) [inline system-variables.md:4157]:
```suggestion
- 默认值：`""`
- 可选值：`""`，`TABLE`，`GLOBAL`
```

File: system-variables.md (hunk 12 above)
Comment (hfxsd) [inline system-variables.md:4159]:
```suggestion
    - 默认值为空 `""`：表示对于表 Region 不进行打散。
```

File: system-variables.md [hunk 13]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 类型：布尔型
-- 默认值：`OFF`
-- TiDB 默认会在建表时为新表分裂 Region。开启该变量后，会在建表语句执行时，同步打散刚分裂出的 Region。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。
+- 类型：枚举型
+- 默认值：`""`
+- 可选值：`""`，`TABLE`，`GLOBAL`
+- TiDB 默认会在建表时为新表分裂 Region，使用该变量控制表分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，能让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值包括：
```
Comment (lilin90) [inline system-variables.md:4158]:
```suggestion
- TiDB 默认会在建表时为新表分裂 Region。该变量用于控制表的分裂打散策略，TiDB 会根据选择的打散策略进行 Region 打散。适用于批量建表后紧接着批量写入数据，让刚分裂出的 Region 先在 TiKV 分散而不用等待 PD 进行调度。为了保证后续批量写入数据的稳定性，建表语句会等待打散 Region 完成后再返回建表成功，建表语句执行时间会是该变量关闭时的数倍。可选值描述如下：
```

File: system-variables.md [hunk 14]
```diff
@@ -4149,12 +4149,16 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+    - `""`：默认值，表示不打散表 Region。
+    - `TABLE`：表示在建表时，如果设置了 `PRE_SPLIT_REGIONS` 或者 `SHARD_ROW_ID_BITS` 属性，预分裂多个 Region 的场景下，会按表的粒度对这些表的 Region 进行打散。但是如果在建表时没有设置上述属性，需要快速创建大量表的场景，会导致这些表的 Region 集中在其中几个 TiKV 节点上，造成 Region 分布不均匀。
+    - `GLOBAL`：表示 TiDB 会根据整个集群的数据分布情况来打散新建表的 Region。特别是快速创建大量表的时候，使用 `GLOBAL` 可以有效避免 Region 过度集中在少数几个 TiKV 节点上，确保 Region 分布均匀。
```
Comment (lilin90) [inline system-variables.md:4161]:
```suggestion
    - `GLOBAL`：表示 TiDB 会根据整个集群的数据分布情况来打散新建表的 Region。特别是快速创建大量表的时候，使用 `GLOBAL` 可以有效避免 Region 过度集中在少数几个 TiKV 节点上，确保 Region 在集群中分布均匀。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by hfxsd — APPROVED** [review]

**Pull request merged**